pennylane_lightning/core/src/simulators/lightning_gpu/algorithms/AdjointJacobianGPUMPI.hpp

```diff
@@ -235,6 +238,8 @@ class AdjointJacobianMPI final
         if (!jd.hasTrainableParams()) {
             return;
         }
+        MPIManager mpi_manager_(jd.getMPIManager());
```

Comment:
Is there a need for all of the blocking calls here? Or is this the required fix from earlier? What PR added these lines?

Reply:
Not really. Only few lines needed. Will update it later after retro.